src/stage0.txt

```diff
@@ -12,14 +12,14 @@
 # source tarball for a stable release you'll likely see `1.x.0` for rustc and
 # `0.x.0` for Cargo where they were released on `date`.
 
-date: 2019-12-18
-rustc: beta
-cargo: beta
+date: 2020-01-27
+rustc: 1.41.0
+cargo: 0.42.0
 
 # We use a nightly rustfmt to format the source because it solves some bootstrapping
 # issues with use of new syntax in this repo. If you're looking at the beta/stable branch, this key should be omitted,
 # as we don't want to depend on rustfmt from nightly there.
-rustfmt: nightly-2019-12-18
+#rustfmt: nightly-2019-12-18
```

Comment:
I'm really not sure but this may cause the failure here? (Yeah, the comment says it should be omitted though.)

Reply:
No, this should not be responsible. The rustfmt here is purely for x.py fmt, it doesn't affect testing at all (we build a rustfmt from scratch from that).

Reply:
Oh, didn't know that. So it shouldn't be the cause...